helpers/aws-lc/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2009 The Go Authors. All rights reserved.
+
```

Comment:
Keeping this file as LICENSE makes sense given the references in the copied code; but add the project's copyright here too:

`Copyright Amazon.com, Inc., its affiliates, or other contributors.`

Given the scope of this code, having our changes be under BSD-3-Clause rather than Apache-2.0/MIT makes sense.